src/main/java/net/kodehawa/mantarobot/commands/HelpCmd.java

```diff
@@ -564,9 +518,35 @@ private void buildHelp(Context ctx, CommandCategory category) {
         );
     }
 
-    // Transitional command.
-    @Subscribe
-    public void slash(CommandRegistry cr) {
+    // This looks hilarious.
+    // Maybe just remove it.
+    @Alias("info") @Alias("status") @Alias("shard")
+    @Alias("shardinfo") @Alias("ping") @Alias("time")
+    @Alias("prune") @Alias("ban") @Alias("kick")
+    @Alias("softban") @Alias("userinfo") @Alias("serverinfo")
+    @Alias("avatar") @Alias("roleinfo") @Alias("support")
+    @Alias("donate") @Alias("language") @Alias("invite")
+    @Alias("danbooru") @Alias("e621") @Alias("e926")
+    @Alias("yandere") @Alias("konachan") @Alias("gelbooru")
+    @Alias("safebooru") @Alias("rule34") @Alias("iam")
+    @Alias("iamnot") @Alias("8ball") @Alias("createpoll")
+    @Alias("anime") @Alias("character") @Alias("poll")
+    @Alias("coinflip") @Alias("ratewaifu") @Alias("roll")
+    @Alias("birthday") @Alias("profile") @Alias("reputation")
+    @Alias("equip") @Alias("unequip") @Alias("badges")
+    @Alias("activatekey") @Alias("premium") @Alias("transfer")
+    @Alias("itemtransfer") @Alias("pet") @Alias("waifu")
+    @Alias("play") @Alias("shuffle") @Alias("np")
+    @Alias("skip") @Alias("stop") @Alias("ns")
+    @Alias("volume") @Alias("forceplay") @Alias("lyrics")
+    @Alias("playnow") @Alias("rewind") @Alias("forward")
+    @Alias("restartsong") @Alias("removetrack") @Alias("move")
+    @Alias("cast") @Alias("salvage") @Alias("repeat")
+    @Alias("repair") @Alias("marry") @Alias("divorce")
+    @Alias("mute") @Alias("unmute") @Alias("remindme")
+    @Alias("game") @Alias("trivia") @Alias("love")
+    @Category(CommandCategory.HIDDEN)
+    public static class Slash extends TextCommand {
```

Comment:
In all 100% serious do we really still need this?

Reply:
Removed, it's not